src/test/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConnectorConfigBuilder.java

```diff
@@ -0,0 +1,75 @@
+package com.marklogic.kafka.connect.sink;
+
+import org.apache.kafka.connect.runtime.ConnectorConfig;
+
+import java.util.Properties;
+import java.util.UUID;
+
+public class MarkLogicSinkConnectorConfigBuilder {
+
+    private String topic = String.format("topic-%s", UUID.randomUUID().toString());
+
+    private String key = String.format("key-%s", UUID.randomUUID().toString());
+
+    private final Properties connectorProps = new Properties();
+
+    public MarkLogicSinkConnectorConfigBuilder withTopic(final String topic) {
+        this.topic = topic;
+        return this;
+    }
+
+    public MarkLogicSinkConnectorConfigBuilder withKey(final String key) {
+        this.key = key;
+        return this;
+    }
+
+    public <T> MarkLogicSinkConnectorConfigBuilder with(final String propertyName, final T value) {
+        connectorProps.put(propertyName, value);
+        return this;
+    }
+
+    public <T> MarkLogicSinkConnectorConfigBuilder withAll(final Properties connectorProps) {
+        this.connectorProps.putAll(connectorProps);
+        return this;
+    }
+
+    private <T> void ifNonExisting(final String propertyName, final T value) {
+        if (connectorProps.get(propertyName) != null) return;
+        connectorProps.put(propertyName, value);
+    }
+
+    public Properties build() {
+
+        ifNonExisting(ConnectorConfig.NAME_CONFIG, "marklogic-sink");
+        ifNonExisting(ConnectorConfig.CONNECTOR_CLASS_CONFIG, "MarkLogicSinkConnector");
+        ifNonExisting(ConnectorConfig.TASKS_MAX_CONFIG, "1");
+        ifNonExisting("topics", topic);
+        ifNonExisting("key", key);
+
+        ifNonExisting("ml.connection.host", "localhost");
```

Comment:
Use the constants in MarkLogicConfig here, just to avoid duplicating the strings (which the tests aren't yet consistent about, but that's a gradual cleanup effort).